src/geocat/viz/util.py

```diff
@@ -157,8 +156,7 @@ def add_right_hand_axis(ax,
         y_minor_per_major (:class:`int`):
             Number of minor ticks between adjacent major ticks on y-axis.
 
-        Returns
-        -------
+    Returns:
```

Comment:
I would actually suggest using the original method here, as it more closely aligns with numpydoc style and would be more consistent with other geocat repos. I don't believe geocat-viz ever got the docstring overhauls that geocat-comp and uxarray got, which might partially explain the internal formatting differences.